src/wp-includes/block-supports/layout.php

```diff
@@ -6,16 +6,204 @@
  * @since 5.8.0
  */
 
+
+/**
+ * Returns layout definitions, keyed by layout type.
+ *
+ * Provides a common definition of slugs, classnames, base styles, and spacing styles for each layout type.
+ * When making changes or additions to layout definitions, the corresponding JavaScript definitions should
+ * also be updated.
+ *
+ * @since 6.3.0
+ * @access private
+ *
+ * @return array[] Layout definitions.
+ */
+function wp_get_layout_definitions() {
```

Comment:
Typically private functions are prefixed with an `_` in an attempt to discourage their use by extenders.

If plugin or theme developers will need it then I reckon it can be left public and the `@access` annotation removed.

Reply:
Ignore this, Felix made a good point here. I gather the background is that it's not uncommon that functions get the private delegation removed so the `_` prefix includes a few public functions. https://github.com/WordPress/wordpress-develop/pull/4614#discussion_r1232519119